### ISSUE TITLE
Request: Add Documentation field support

### ISSUE THREAD

**Issue by megaroeny** [madebyfabian/figma-plugin-bunch-description-change#8]
Would be awesome if you could also add the same functionality, for the Documentation field. Thanks!

![image](https://user-images.githubusercontent.com/1559447/111506237-d0b21600-871f-11eb-8609-b3ac0e53f598.png)

**Comment by madebyfabian** [comment]
Great idea! But unfortunatly for now, the Figma Plugin API doesn't allow plugins to access this field. https://www.figma.com/plugin-docs/api/ComponentNode/#docsNav

If they will, I'll consider adding this functionality ^^

**Comment by megaroeny** [comment]
> Great idea! But unfortunatly for now, the Figma Plugin API doesn't allow plugins to access this field. https://www.figma.com/plugin-docs/api/ComponentNode/#docsNav
> 
> If they will, I'll consider adding this functionality ^^

Ahhh! Sorry I wasn't aware, been a while since I got my hands into the API. I'll leave this open for you in the future. 😄

**Comment by madebyfabian** [comment]
Hey there @megaroeny!
Figma actually released an update of their' plugins API: https://www.figma.com/plugin-docs/blog/2021/04/28/version-1-update-23/ 
wich allows to modify the field you've requested. So. Long story short, I added support for it, be sure to reload figma to get the newest version of the plugin :)

But, as Figma sees this field as a URL, two things that could occur: If the end result is not a url, you won't be able to change it. (it will throw you an error message). Also, if you want to clear this field, every field gets "http://" because that's Figma's default value, and there is nothing I can do about it to "really" clear it. I added an issue here: https://github.com/figma/plugin-typings/issues/33 so I will still keep this opened.
